code/game/objects/items.dm

```diff
@@ -630,3 +637,30 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 /obj/item/proc/microwave_act(obj/machinery/microwave/M)
 	if(M && M.dirty < 100)
 		M.dirty++
+
+/obj/item/proc/friendly_force()
+	if(force)
+		if(force <= 3)
+			f_force = "very low"
+		else if(force <= 5)
+			f_force = "low"
+		else if(force <= 7)
+			f_force = "medium"
+		else if(force <= 10)
+			f_force = "<font color=green>high</font>"
+		else if(force <= 15)
+			f_force = "<font color=red>Combat Grade</combat>"
+		else if(force <= 24)
+			f_force = "<font color=orange>high Combat Grade</font>"
+		else if(force >= 25)
+			f_force = "<font color=white>Exceptionally Lethal</font>"
+
+
+/obj/item/MouseEntered(location,control,params)
+	if(held)
+		friendly_force()

```

Comment:
why store it if you're going to computer it again every time?
What you should do is store both ```force_string``` (yes, rename ```f_force```) and also store the force it had when you last ran ```update_force_string()``` (yes, renamed), then here you can run it again, eg:
```
if(last_force_string_check != force)
    update_force_string()
```